parachain/primitives/core/src/pricing.rs

```diff
@@ -23,18 +24,20 @@ pub struct Rewards<Balance> {
 	pub remote: U256,
 }
 
-#[derive(RuntimeDebug)]
+#[derive(Clone, PartialEq, RuntimeDebug)]
 pub struct InvalidPricingParameters;
 
 impl<Balance> PricingParameters<Balance>
 where
 	Balance: BaseArithmetic + Unsigned + Copy,
 {
 	pub fn validate(&self) -> Result<(), InvalidPricingParameters> {
-		if self.exchange_rate == FixedU128::zero() {
+		if self.exchange_rate < FixedU128::from_rational(1, 10000) ||
+			self.exchange_rate > FixedU128::from_rational(1, 1)
+		{
```

Comment:
I prefer not to change the validation logic. I also thought about this approach, but ultimately decided it wasn't worth it. 

1. If the exchange rate does actually mistakenly get set to `1/9999` (slightly less than the limit), then that practically still ends up blocking the bridge anyway.
2. The upper limit is arbitrary, and if the real-world exchange rate breaches the limit (quite possible), then a runtime upgrade would be required to fix the validation code.